src/app/containers/Video/index.jsx

```diff
@@ -12,15 +14,27 @@ import { filterForBlockType } from '../../helpers/blockHandlers';
 const VideoContainer = ({ blocks }) => {
   const captionBlock = filterForBlockType(blocks, 'caption');
   const aresMediaBlock = filterForBlockType(blocks, 'aresMedia');
+  const metadata = videoMetadata(aresMediaBlock);
   if (!aresMediaBlock) {
     return null;
   }
 
   return (
-    <Figure>
-      <Video />
-      {captionBlock ? <Caption block={captionBlock} video /> : null}
-    </Figure>
+    <>
+      {metadata ? (
+        <Helmet>
+          {
+            <script type="application/ld+json">
```

Comment:
```suggestion
            <script type="application/json+ld">
```
@Bopchy should it be `json+ld` or `ld+json`.. does the distinction matter? I don't know but it's just something I noticed while reviewing..

Reply:
It should be `application/ld+json`. It's correct in the PR. https://developers.google.com/search/docs/guides/intro-structured-data

Reply:
sweet! thanks 😄